bin/generate-readme

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+transcribe() {
+  local -r project="$(node --print 'require("./package.json").repository.url.replace(/[.]git$/, "").split("/").slice(-2).join("/")')"
+  local -r version="${VERSION?Environment variable not set}"
+  local -r url="https://github.com/$project/blob/v$version/{filename}#L{line}"
```

Comment:
I see we rely explicitly on version being prefixed by `v`. I don't see a script for XYZ specified in this repo, so this means that one has to configure XYZ properly for `sanctuary-scripts` to work. Maybe we can add an `release.sh` or something to this repo which already ensures the correct tagging convention? Or if not, we should add a note to the README.